src/app/beer_garden/monitor.py

```diff
@@ -107,7 +107,8 @@ def __init__(
             self.observer.start()
 
     def on_created(self, event):
-        """ When a user VIM edits a file it DELETES, then CREATES the file, this captures that use case"""
+        """When a user VIM edits a file it DELETES, then CREATES the file, this
+        captures that use case"""
```

Comment:
This actually violates a PEP: https://www.python.org/dev/peps/pep-0257/#id16. Specifically, if you're going to use a oneline docstring then it should really be one line.

However, more importantly, this is good information to have in this method's docstring. It tells me why we bother to implement this at all, which is important since my first impression was "why are we doing this, we should never allow them to just create a config file in the middle of running." But that info isn't really appropriate for a summary line (the first line in a multi-line docstring), that line should be used to describe the function itself.

 So something like this would be preferable:

> """Callback invoked when the file is created
>
> Some text editors (like VIM) will delete and create a file during interactive editing. Implementing this method allows for capturing this case."""